h/services/annotation_metadata.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+from sqlalchemy import func, select
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import Session
+
+from h.models import Annotation, AnnotationMetadata, AnnotationSlim
+from h.security import decrypt_jwe_dict
+
+
+class AnnotationMetadataService:
+    def __init__(self, db: Session, secret: str):
+        self._db = db
+        self._secret = secret
+
+    def set_annotation_metadata_from_jwe(self, annotation: Annotation, jwe: str):
```

Comment:
```suggestion
    def set(self, annotation: Annotation, jwe: str):
```

I think we're gonna end up saying "annotation metadata" and "jwe" each twice at call time (actually `annotation` three times):

```python
annotation_metadata_svc.set_annotation_metadata_from_jwe(annotation, jwe)
```

I reckon it's unnecessary to include `annotation_metadata` in the method name: this is already `AnnotationMetadataService` so it's clear that it will be setting annotation metadata.

I reckon we can remove the `jwe` from the name as well. If we ever add support for setting annotation metadata from things other than JWTs, then we can append the `from_jwt` to the method name at that point, or maybe we can get clever and try to use `@functools.singledispatch`.

```python
annotation_metadata_svc.set(annotation, metadata_jwe)
```